network/openvswitch_bridge.py

```diff
@@ -80,10 +95,15 @@ class OVSBridge(object):
     def __init__(self, module):
         self.module = module
         self.bridge = module.params['bridge']
+        self.parent = module.params['parent']
+        self.vlan = module.params['vlan']
         self.state = module.params['state']
         self.timeout = module.params['timeout']
         self.fail_mode = module.params['fail_mode']
 
+        if self.parent and not self.vlan:
+            self.module.fail_json(msg='VLAN id must be set when parent is defined')
+
```

Comment:
@stygstra I was letting OpenVswicht do this kind of test. But it a nice to have, so I added it in e4c2e954865d520354c617c3cdc83e557b3a0c12